### PR TITLE
Minify images using ImageOptim (lossless compression)

### PR DESCRIPTION
**Pull request by vraravam**
Second try. This time around, the script itself reports the size before and after processing. Based on this, the number of files touched is much smaller. I have also manually verified each touched image by opening it and checking that it has some image (and is not a zero-byte file).

@kytwb - please review.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "package": "node scripts/package.js",
-    "create": "node scripts/create.js"
+    "create": "node scripts/create.js",
+    "minify-images": "./minify-images.sh"
```
Comment (kytwb) [inline package.json:12]: Should we move this in the `scripts` folder?

Comment (vraravam) [reply package.json:12]: sure - i can do that @kytwb

Comment (vraravam) [reply package.json]: Done in the amended commit @kytwb

**Comment by kytwb** [comment]
@vraravam thanks for the iteration!

**Pull request merged**